interfaces/naming.go

```diff
@@ -44,3 +46,17 @@ func WrapperNameForApp(snapName, appName string) string {
 func SecurityTagForApp(snapName, appName string) string {
 	return fmt.Sprintf("%s.snap", WrapperNameForApp(snapName, appName))
 }
+
+// SecurityTag returns application-specific security tag.
```

Comment:
Both this and the above function claim to offer an app-specific security tag, and they disagree about how it should look like.

We also need to get rid of WrapperNameForApp. This is not the place this convention should be defined.. (we're handling interfaces here).